components/family-studies/rest/src/main/java/org/phenotips/studies/family/rest/internal/DefaultFamiliesResourceImpl.java

```diff
@@ -182,7 +182,9 @@ public Response listFamilies(Integer start, Integer number, Boolean fullJSON)
                         } else {
                             familyJSON = getFamilySummaryJSON(record, this.uriInfo);
                         }
-                        familyList.put(familyJSON);
+                        if (familyJSON != null) {
```

Comment:
Now this breaks checkstyle, too many branches in this method. Can you look at it?